Makefile

```diff
@@ -113,6 +113,10 @@ docs-watch: docs-preview
 docs-clean:
 	rm -r docs_build
 
+.PHONY: docs-examples
+docs-kustomize-examples: yq kustomize ## Generate updated docs from examples using kustomize
```

Comment:
I think you can add it here:
```
.PHONY: docs
docs: manifests docs-dependencies docs-kustomize-examples crd-to-markdown ## Build docs
```
https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/Makefile#L96-L97

Reply:
That makes sense.